lib/hammer_cli_katello/repository.rb

```diff
@@ -441,6 +447,16 @@ def print_results(name, results)
           end
         end
       end
+
+      def silence_warnings
+        original_verbose = $VERBOSE
```

Comment:
where does $VERBOSE come from?

Reply:
Directly in Ruby, see https://mislav.net/2011/06/ruby-verbose-mode/ for some more details. Basically, it suppresses the output from `warn` calls.